app/templates/suppliers/view_esignature_agreement.html

```diff
@@ -0,0 +1,64 @@
+{% extends "_base_page.html" %}
+{% import "toolkit/summary-table.html" as summary %}
+
+{% block pageTitle %}
+  {{ framework.name }} agreement for {{ supplier_framework.declaration.nameOfOrganisation }} – Digital Marketplace admin
+{% endblock %}
+
+{% block breadcrumbs %}
+  {{ govukBreadcrumbs({
+    "items": [
+      {
+        "text": "Admin home",
+        "href": url_for('.index')
+      },
+      {
+        "text": "{} agreements".format(framework.name),
+        "href": url_for('.list_agreements', framework_slug=framework.slug)
+      },
+      {
+        "text": "Framework agreement"
+      }
+    ]
+  }) }}
+{% endblock %}
+
+{% block mainContent %}
+  <h1 class="govuk-heading-l">{{ company_details.registered_name }}</h1>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+        <p><a href="{{ url_for('.supplier_details', supplier_id=supplier.id) }}">View company details</a></p>
+
+       <h2 class="govuk-heading-m">Signature details</h2>
+        {% call(item)
+              summary.list_table(
+                [
+                  {"name": "Appointment is to", "value": ", ".join(lot_names) },
```

Comment:
Can we use jinja syntax here e.g. `lot_names | join(', ')`?

Reply:
We can indeed!